api/views/education_documentation.html

```diff
@@ -0,0 +1 @@
+<html><h1>Education Documentation</h1></html>
```

Comment:
I like that we are ready to expose documentation directly through the be server "Website"